source/ed/connectors/gtfs_parser.cpp

```diff
@@ -1351,7 +1342,42 @@ boost::gregorian::date_period GenericGtfsParser::find_production_date(const std:
         }
 
     }
+    return complete_production_date(beginning_date, start_date, end_date);
+}
+
+void GenericGtfsParser::manage_production_date(Data& data, const std::string& beginning_date){
+    auto start_it = data.feed_infos.find("feed_start_date");
+    auto end_it = data.feed_infos.find("feed_end_date");
+    boost::gregorian::date start_date(boost::gregorian::not_a_date_time), end_date(boost::gregorian::not_a_date_time);
+    if ((start_it != data.feed_infos.end()) && (end_it != data.feed_infos.end())) {
+        try{
+            start_date = boost::gregorian::from_undelimited_string(start_it->second);
+        }catch(const std::exception& e) {
+           LOG4CPLUS_WARN(logger, "manage_production_date, Unable to parse start_date :"<<start_it->second<<", error : "<<std::string(e.what()));
```

Comment:
I don't think the string cast is necessary, it should be enough to do `<< e.what()`

you need also to take of the spacing, put some spaces around the `<<`  and `{`